pkg/postage/stampissuer_test.go

```diff
@@ -40,3 +42,59 @@ func newTestStampIssuer(t *testing.T, block uint64) *postage.StampIssuer {
 	}
 	return postage.NewStampIssuer("label", "keyID", id, big.NewInt(3), 16, 8, block, true)
 }
+
+func Test_StampIssuer_inc(t *testing.T) {
+	t.Parallel()
+
+	addr := swarm.NewAddress([]byte{1, 2, 3, 4})
+
+	t.Run("mutable", func(t *testing.T) {
+		t.Parallel()
+
+		sti := postage.NewStampIssuer("label", "keyID", make([]byte, 32), big.NewInt(3), 16, 8, 0, false)
+		count := sti.BucketUpperBound()
```

Comment:
I would suggest you cast `count` here and then you don't have to do it in the loops below. The same is for the other subtests.